x/wasm/keeper/options.go

```diff
@@ -21,23 +21,39 @@ func WithWasmEngine(x types.WasmerEngine) Option {
 }
 
 // WithMessageHandler is an optional constructor parameter to set a custom handler for wasmVM messages.
-// This option should not be combined with Option `WithMessageEncoders`.
+// This option should not be combined with Option `WithMessageEncoders` or `WithMessageHandlerDecorator`
```

Comment:
Go documentation does not support any markup language. I think wrapping code in back quotes (`) is not very common, since it does not do anything. 

Since each public token comment starts with that token name (here `WithMessageHandler`), it is inconsistent that the first occurrence is not surrounded by back quotes and further code mentions are.

:nail_care: